File: py/Soal-2/gajah.py
from mamalia import Mamalia

class Gajah(Mamalia) :
    def __init__(self) :
        super().__init__()
        super().setNama("Gajah")
        super().setUkuran("Besar")
        super().setJumlahKaki(4)
        super().setHabitat("Padang rumput")

File: cpp/Soal-2/gajah.cpp
#include "mamalia.cpp"
#pragma once

class gajah: public mamalia
{
public:
    gajah() {
        mamalia::setNama("Gajah");
        mamalia::setUkuran("Besar");
        mamalia::setJumlahKaki(4);
        mamalia::setHabitat("Padang rumput");
    }
};